syntax = "proto2";

package experience_replay;

message Trajectory {
    repeated int32 visual_obs = 1 [packed=true];
    repeated float vector_obs = 2 [packed=true];
    repeated int32 actions = 3 [packed=true];
    repeated float rewards = 4 [packed=true];
}

message Info {
    repeated int32 visual_obs_spec = 5 [packed=true];
    repeated int32 vector_obs_spec = 6 [packed=true];
    optional int32 action_spec = 7;
    optional int32 num_steps = 8;
}